app/subcommands/stop_subcommand.py

```diff
@@ -91,5 +91,9 @@ def _terminate_running_procs(self, procs_to_termintate):
 
     def _kill_running_procs(self, procs_to_kill):
         for proc in [p for p in procs_to_kill if p.is_running()]:
-            self._logger.info("Sending SIGKILL to PID {}", proc.pid)
-            os.kill(proc.pid, signal.SIGKILL)
+            if 'SIGKILL' in dir(signal):
```

Comment:
What do you think of changing this to `if hasattr(signal, 'SIGKILL'):` instead? [This SO question](http://stackoverflow.com/questions/17723569/whats-the-difference-between-hasattr-and-attribute-in-dir) makes me think that it might the preferred approach.